docs/modules/ROOT/pages/tutorials/demo-app/appcat-nextcloud.adoc

```diff
@@ -0,0 +1,128 @@
+the https://console.cloudscale-lpg-2.appuio.cloud/command-line-tools[OpenShift console command line tools].
+Then you can show the command to create your kubeconfig in the https://oauth-openshift.apps.cloudscale-lpg-2.appuio.cloud/oauth/token/display?code=sha256~xhC3o8qafyrrISMXZhKRt9uhIf-687n7LCHL6jpYl5k&state=[OpenShift console].
+
+.Generate kubeconfig for pipeline
+[source,shell]
+----
+KUBECONFIG=./config-for-pipeline
+oc login ...
+cat $KUBECONFIG
+----
+
+Now you need to add this kubeconfig to the GitHub *secrets*. You find them under `Settings → Environments → choose environment → (+) Add secret`. In this case, the environment name is `prod`.
+
+We now have the pipeline and a minimal claim. But there are some steps missing
+to make it production ready.
+
+== Make it Production Ready
+
+To make Nextcloud production ready, you need to configure the following settings:
+
+.VSHNNextcloud: encrypted disk
+[source,yaml]
+----
+apiVersion: vshn.appcat.vshn.io/v1
+kind: VSHNNextcloud
+metadata:
+  name: appcat-demo-nextcloud-appuio-simple
+  namespace: vshn2-demo-nextcloud-appuio-simple-prod
+spec:
+  parameters:
+    backup:
+      retention:
+        keepDaily: 6 # <1>
+    instances: 1 # <2>
+    maintenance: # <3>
+      dayOfWeek: wednesday
+      timeOfDay: '23:42:18'
+    monitoring:
+      email: your@email.com # <4>
+    service:
+      fqdn:
+        - appuio-nextcloud-demo.apps.cloudscale-lpg-2.appuio.cloud
+      version: "32"
+      postgreSQLParameters:
+        encryption:
+          enabled: true
+        service:
+          majorVersion: "17" # <5>
+        monitoring:
+          email: your@email.com # <6>
+        instances: 1 # <7>
+    size:
+      plan: standard-2
+      disk: 16Gi # <8>
+  writeConnectionSecretToRef:
+    name: nextcloud-creds
+----
+<1> The number of daily backups to keep.
+<2> Set to a value greater than 1 if you want to run Nextcloud with High Availability and update it without downtime.
+<3> When the maintenance job should run.
+<4> The email address where user alerts for Nextcloud should be sent.
+<5> The major version for the PostgreSQL instance. Start with the most recent major version.
+<6> The email address where user alerts for the PostgreSQL instance are sent.
+<7> Set to a value greater than 1 if you want to run PostgreSQL with High Availability and update it without downtime.
+<8> The desired disk space for your Nextcloud instance. This can only be increased later.
+
+After you have configured all these settings, you can deploy your Nextcloud.
+You will find the credentials to log in to your Nextcloud in the secret you specified previously.
+You can retrieve this secret using the OpenShift console under https://console.cloudscale-lpg-2.appuio.cloud/k8s/all-namespaces/core~v1~Secret[Workloads > Secret] (Here for cloudscale).
+If you used the provided link, you will need to choose the correct project in the dropdown menu.
```

Comment:
Mention that it takes a while until the nextcloud is available...and even more time until the secret is available.

Reply:
done